include/zephyr/drivers/espi.h

```diff
@@ -242,17 +242,21 @@ enum espi_vwire_signal {
 	ESPI_VWIRE_SIGNAL_SLV_GPIO_5,
 	ESPI_VWIRE_SIGNAL_SLV_GPIO_6,
 	ESPI_VWIRE_SIGNAL_SLV_GPIO_7,
-
-	/* USB-C port over current */
-	ESPI_VWIRE_SIGNAL_OCB_0,
-	ESPI_VWIRE_SIGNAL_OCB_1,
-	ESPI_VWIRE_SIGNAL_OCB_2,
-	ESPI_VWIRE_SIGNAL_OCB_3,
+	ESPI_VWIRE_SIGNAL_SLV_GPIO_8,
```

Comment:
is Windows using ESPI_VWIRE_SIGNAL_SLV_GPIO_8 and above? if not we can remove these

Reply:
No, but since generic custom VWs mechanism is part of this PR scope, I see no harm

Reply:
I don't a have strong opinion, but if unused it looks like we are spamming header file

Reply:
I agree with @albertofloyd that it is no harm to keep them; I don't see it as spamming the header file; it may have use later

Reply:
ACK